src/components/AuctionFooter/index.tsx

```diff
@@ -58,9 +58,7 @@ const AuctionFooter: React.SFC<AuctionFooterProps> = ({
   }
 
 
-  return (
-    <div className="auctionFooter"/>
-  )
+  return null
```

Comment:
Consider a gradual footer removal rather than it just disappearing from under you.